### ISSUE TITLE
Add initial workflow step

### ISSUE THREAD

**Issue by rogerhutchings** [zooniverse/Bat-Detective#60]
First question should be "Is there anything interesting?", which counts as a classification if no.

**Comment by macaodha** [comment]
Yes, something like "Does this clip contain a bat, insect, or machine?" with a yes/no option would be great.
